mypy/build.py

```diff
@@ -878,19 +839,15 @@ def find() -> Optional[str]:
         # that will require the same subdirectory.
         components = id.split('.')
         dir_chain = os.sep.join(components[:-1])  # e.g., 'foo/bar'
-        if (dir_chain, lib_path) not in find_module_dir_cache:
+        if (dir_chain, lib_path) not in cache.dirs:
             dirs = []
             for pathitem in lib_path:
                 # e.g., '/usr/lib/python3.4/foo/bar'
-                isdir = find_module_isdir_cache.get((pathitem, dir_chain))
```

Comment:
Why was this block of code removed?

Reply:
It's maintaining an `isdir` cache, which is handled by the `fscache` stat cache.